documentation/samples/packs/todoist/full.ts

```diff
@@ -397,92 +394,59 @@ pack.addFormula({
 });
 
 pack.addFormula({
```

Comment:
As an alternative to excluding due date from 2-way, what do you think of either:
- Treating a time of midnight as a date without a time, or
- Splitting it across two columns so there's a datetime column and an `endOfDate` column

Either of those approaches could allow for updating with 2-way without the additional action formula

Reply:
> Treating a time of midnight as a date without a time

I thought about this, but it would require editors and viewers to understand a fairly subtle behavioral difference between 12:00 and 12:01.

> Splitting it across two columns so there's a datetime column and an `endOfDate` boolean/checkbox column

I tried something similar (`dueDisplay`, `dueDate`, `dueDateTime`) and it didn't feel like a great UX. Plus Todoist's system for due dates is actually even more complex, allowing for recurrences, etc. I chose to omit that for brevity, but all said I think the correct way to display this complex bundle of information is as a sub-object. Artificially flattening it out into N different columns helps with editing, but hurts consuming, IMO.

I imagine that is circumstance (having a rich object you want to allow users to modify in some ways) will be somewhat common, so I think there is also some utility in demonstrating how to handle that case. Assuming we think this is how we would recommend developers solve that case.